addons/mail/static/tests/discuss/core/web/sidebar_tests.js

```diff
@@ -15,7 +19,9 @@ QUnit.test("sidebar find shows channels matching search term", async () => {
     });
     const { openDiscuss } = await start();
     openDiscuss();
-    await click(".o-mail-DiscussSidebarCategory-add:eq(0)");
+    await click(
+        ":nth-child(1 of .o-mail-DiscussSidebarCategory) .o-mail-DiscussSidebarCategory-add"
+    );
```

Comment:
Too bad CSS selector is more strict than Sizzle one... It's less DRY, makes test less readable :(